.github/workflows/pull-request.yml

```diff
@@ -4,8 +4,8 @@ on:
     types:
       - opened
       - synchronize
-    branches:
-      - main
+    branches-ignore:
+      - production
```

Comment:
This change was to allow this PR to target a non-master branch. That branch has since merged but I think this is still a useful addition.